crates/bindings-csharp/Codegen/Module.cs

```diff
@@ -394,21 +467,24 @@ public void Initialize(IncrementalGeneratorInitializationContext context)
             .WithTrackingName("SpacetimeDB.Reducer.GenerateClass")
             .Collect();
 
+        var tableViews = tables
+            .SelectMany((t, ct) => t.GenerateViews())
+            .WithTrackingName("SpacetimeDB.Table.GenerateViews")
+            .Collect();
+
         context.RegisterSourceOutput(
-            tableNames.Combine(addReducers),
+            tableViews.Combine(addReducers),
             (context, tuple) =>
             {
                 // Sort tables and reducers by name to match Rust behaviour.
                 // Not really important outside of testing, but for testing
                 // it matters because we commit module-bindings
                 // so they need to match 1:1 between different langs.
-                var tableNames = tuple.Left.Sort();
+                var tableViews = tuple.Left.Sort((a, b) => a.Key.viewName.CompareTo(b.Key.viewName));
                 var addReducers = tuple.Right.Sort((a, b) => a.Key.CompareTo(b.Key));
                 // Don't generate the FFI boilerplate if there are no tables or reducers.
-                if (tableNames.IsEmpty && addReducers.IsEmpty)
-                {
```

Comment:
Why did these get removed? I thought we even have mandatory braces in editorconfig so it should tell you about style mismatch 🤔

Reply:
Huh, didn't even realize I removed these, I'll add them back